<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SeedToTime3</class>
 <widget class="QMainWindow" name="SeedToTime3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>355</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>504</width>
    <height>355</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>504</width>
    <height>355</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gen 3 Seed to Time</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QTabWidget {
	background-color: #274050;
	border-radius: 10px;
	margin-top: 7px;
	font-family: &quot;Myriad Pro&quot;, Montserrat;
	font-weight: 500;
}

QPushButton {
	border: none;
	border-radius: 6px;
	color: white;
	font-family: &quot;Myriad Pro&quot;, Montserrat;
	font-weight: 500;
}

QLabel {
	color: black;
	font-family: &quot;Myriad Pro&quot;, Montserrat;
	font-weight: 500;
}

QWidget QLineEdit {
	font-family: &quot;Myriad Pro&quot;, Montserrat;
	font-weight: 500;
}

QLineEdit {
	qproperty-alignment: AlignCenter;
	background-color: #1D303C;
	border:none;
	border-radius: 6px;
	color: white;
}

QTableView {
	border: 4px solid DodgerBlue;
}

QHeaderView::section {
	border: none;
	background-color: #274050;
	padding-top: 3px;
	padding-bottom: 3px;
	color: white;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="seedLabel">
        <property name="text">
         <string>16/32-Bit Seed</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QTextBox" name="seedToTimeSeed">
        <property name="maxLength">
         <number>8</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="descriptionLabel1">
        <property name="text">
         <string>Enter a 16-bit seed</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="yearLabel">
        <property name="text">
         <string>Year</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QTextBox" name="seedToTimeYear">
        <property name="text">
         <string notr="true">2000</string>
        </property>
        <property name="maxLength">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="descriptionLabel2">
        <property name="text">
         <string>Or 32-bit seed for closest 16-bit match</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QLabel" name="descriptionLabel3">
        <property name="text">
         <string>This feature only works for Ruby/Sapphire with a live battery</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="pushButtonFind">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>23</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: DodgerBlue;
}</string>
        </property>
        <property name="text">
         <string>Find</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QTableView" name="tableViewGenerator">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>100</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>20</number>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>20</number>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>504</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QTextBox</class>
   <extends>QLineEdit</extends>
   <header>Forms/QTextBox.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
